javalib/src/main/scala/java/io/Throwables.scala

```diff
@@ -21,3 +21,7 @@ class UnsupportedEncodingException(s: String) extends IOException(s) {
 abstract class ObjectStreamException protected (s: String) extends IOException(s) {
   protected def this() = this(null)
 }
+
+class NotSerializableException(s: String) extends ObjectStreamException {
```

Comment:
> It would also be better if you add the NotSerializableException to a new file with its name.

No, actually, we make an exception for, well, exceptions ^^ We put them all in a file `Throwables.scala`, because there are so many of them and they are so tiny that they fill up the list of files in a package for nothing, effectively harming readability.